Update schedule for `GBC 2022 Spring` Semifinal. (#7284)
ppy/osu-wiki: wiki/Tournaments/GBC/GBC_2022_Spring/en.md

| 15 | **GBC最强阵容9个Miyes** | ![][flag_CN] **[miyes](https://osu.ppy.sh/users/12942073)**, ![][flag_CN] [alivn](https://osu.ppy.sh/users/15807665), ![][flag_CN] [ZOM148](https://osu.ppy.sh/users/10402690), ![][flag_CN] [ChiralSS](https://osu.ppy.sh/users/18211706), ![][flag_CN] [1atk](https://osu.ppy.sh/users/20520767) |
| 16 | **头号倒霉蛋** | ![][flag_CN] **[StarTemplar](https://osu.ppy.sh/users/8795096)**, ![][flag_CN] [9961](https://osu.ppy.sh/users/14609209), ![][flag_CN] [1145810](https://osu.ppy.sh/users/24449770), ![][flag_CN] [lochid](https://osu.ppy.sh/users/18107639), ![][flag_CN] [OTruthO](https://osu.ppy.sh/users/17470634) |

> ## Match schedule: Semifinal

Reviewer: ```suggestion
## Match schedule: Semifinals
```

### Saturday, May 7, 2022

| Team A | Team B | Match time |
| --: | :-- | :-: |
| 货拉拉拉不拉拉布拉多 | 太美丽了理塘 | May 7 (Sat) 18:00 (UTC+8) |

### Sunday, May 8, 2022

| Team A | Team B | Match time |
| --: | :-- | :-: |
| GBC最强阵容9个Miyes | 金戈铁马的鲜丰水果摊 | May 8 (Sun) 18:00 (UTC+8) |

## Mappools

> ### Semifinal & Final

Reviewer: ```suggestion
### Semifinals & Finals
```

**[Download the mappack here (130.7 MB)](https://drive.google.com/file/d/1GdRG5ay0LU_RaAfcpml8qG_Gu3bdlWB1/view)**

- Rice
  1. [DJ TOTTO - Chaplet (riunosk) \[Deceptive\]](https://osu.ppy.sh/beatmapsets/1440718#mania/2964130)
  2. [Hachioji-P feat. Hatsune Miku - Blue Star (\[Crz\]Rachel) \[Sparkle Star\]](https://osu.ppy.sh/beatmapsets/1000382#mania/2093764)
  3. [goreshit - satori de pon! (SaltedFISSH) \[\~\~Pon!\~\~\]](https://osu.ppy.sh/beatmapsets/1251545#mania/2601285)

>   4. [DECO*27 - Android Girl feat. Hatsune Miku (Xingyue -) \[AI x1.1\]](https://osu.ppy.sh/beatmapsets/969824#mania/2029268)

Reviewer: ```suggestion
  4. [DECO\*27 - Android Girl feat. Hatsune Miku (Xingyue -) \[AI x1.1\]](https://osu.ppy.sh/beatmapsets/969824#mania/2029268)
```

  7. [Hatsuki Yura - Snow Veil -Shoujo to Kemono no Mori- (Hylotl) \[BAKEMONO\]](https://osu.ppy.sh/beatmapsets/1711963#mania/3498202)
- LN
  1. [DJ TOTTO VS TOTTO - Vajra (IA daisuki) \[IA daisuki&cherrychou's SPECIAL\]](https://osu.ppy.sh/beatmapsets/1677512#mania/3440654)
  2. [YOASOBI - Gunjou (shuniki) \[ShuChan!!+\]](https://osu.ppy.sh/beatmapsets/1296693#mania/2711689)
  3. [HyuN & Ritoru - Apocalypse in Love (IceDynamix) \[Ende der Liebe\]](https://osu.ppy.sh/beatmapsets/925537#mania/1933271)
- SV
  1. [Cutter - EmoCosine (\[GB\]AelSan) \[Miscedence's Cute \[GBC edit\]\]](https://osu.ppy.sh/beatmapsets/1755418#mania/3592340)
  2. [TSAR - 0.0? (\[GB\]AelSan) \[Mafufu & AelSan's @v@!\]](https://osu.ppy.sh/beatmapsets/1755419#mania/3592342)
- Hybrid
  1. [IOSYS - Ringo's Tea Party (TheToaphster) \[Spooked\]](https://osu.ppy.sh/beatmapsets/638838#mania/1355027)
  2. [Camellia feat. Nanahira - Ska-sh All Neurons!! (HowToPlayLN) \[Re-forward !!\]](https://osu.ppy.sh/beatmapsets/1122764#mania/2349915)
- Tiebreaker
  1. **[Camellia - SECRET BOSS (\[GB\]yobrevelc) \[Glorious Battlefield of Champions\]](https://osu.ppy.sh/beatmapsets/1755452#mania/3592441)**

> ### Regular stage week 3 & Quarterfinal

Reviewer: ```suggestion
### Regular stage week 3 & Quarterfinals
```

## Match results

> ### Quarterfinal

Author: ```suggestion
### Quarterfinals
```

### Playoff stage instructions

1. Following the Regular stage, a single elimination stage will be played. This means that the winner continues with their runs, and the losing team will be eliminated from the tournament, except in the bronze match.

### Win conditions

> - In Regular stages and Quarterfinal, teams need to win 6 maps to win a match (best of 11).

Reviewer: ```suggestion
- In the Regular stage and Quarterfinals, teams need to win 6 maps to win a match (best of 11).
```

> - In Semifinal and Final , teams need to win 7 maps to win a match (best of 13).

Reviewer: ```suggestion
- In Semifinals and Finals, teams need to win 7 maps to win a match (best of 13).
```

### Mappool instructions

> 1. There will be three seprate mappools for all stage.

Reviewer: ```suggestion
1. There will be three separate mappools.
```

>    - Regular stage Round 1 and Round 2 use the mappool #1.

Reviewer: ```suggestion
   - Regular stage Round 1 and Round 2 use mappool #1.
```

>    - Regular stage Round 3 and Quarterfinal use the mappool #2.

Reviewer: ```suggestion
   - Regular stage Round 3 and Quarterfinals use mappool #2.
```

>    - Semifinal and Final use the mappool #3

Reviewer: ```suggestion
   - Semifinals and Finals use mappool #3.
```

2. Each mappool consists of a fixed amount of maps each stage which will all be played under FreeMod conditions. This means that there is a unique FreeMod bracket.

> 3. mappool #1 & #2 contain 13 beatmaps, mappool #3 contain 15 beatmaps.

Reviewer: ```suggestion
3. Mappools #1 & #2 contain 13 beatmaps, and mappool #3 contains 15 beatmaps.
```
